hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/TokenFeature.java

```diff
@@ -1093,6 +1131,107 @@ private List<Long> getSerialsForToken(TokenId tokenId) {
                 : nftInfoForToken.stream().map(NftInfo::serialNumber).toList();
     }
 
+    private NetworkTransactionResponse airdropFungibleTokens(
+            TokenId tokenId, int amount, ExpandedAccountId sender, ExpandedAccountId receiver) {
+
+        networkTransactionResponse =
+                tokenClient.executeFungibleTokenAirdrop(tokenId, sender, receiver.getAccountId(), amount);
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
+
+        return networkTransactionResponse;
+    }
+
+    private NetworkTransactionResponse cancelTokenAirdrops(
+            ExpandedAccountId sender, AccountId receiver, TokenId tokenId) {
+        networkTransactionResponse = tokenClient.executeCancelTokenAirdrop(sender, receiver, tokenId);
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
+
+        return networkTransactionResponse;
+    }
```

Comment:
These cancel, claim, airdrop hapi functions are pretty minimal. We can get ride of them completely if we either push assertions into client or add a verify method that wraps the call.

```suggestion
    private NetworkTransactionResponse verify(NetworkTransactionResponse response) {
        assertThat(response.getTransactionId()).isNotNull();
        assertThat(response.getReceipt()).isNotNull();
        networkTransactionResponse = response;
        return response;
    }
```

Reply:
Done - made it that way